### PR TITLE
add the argmax, argmin for the api2.0

### PR DESCRIPTION
**Pull request by wawltor**
### PR types
Others

### PR changes
Others 

### Describe
add the argmax, argmin for the api2.0

### CODE REVIEW

File: python/paddle/tensor/search.py [hunk 1]
```diff
@@ -125,95 +125,168 @@ def argsort(x, axis=-1, descending=False, name=None):
...
+        Tensor, return the tensor of `int32` if set :attr:`dtype` is `int32`, otherwise return the tensor of `int64`
+
+    Examples:
+        .. code-block:: python
+
+            import numpy as np
+            import paddle
+
+            paddle.disable_static()
+            data = np.array([[5,8,9,5],
```
Comment (XiaoguangHu01) [inline python/paddle/tensor/search.py:240]: x = paddle.to_tensor
to_tensor支持list的输入

File: python/paddle/tensor/search.py [hunk 2]
```diff
@@ -125,95 +125,168 @@ def argsort(x, axis=-1, descending=False, name=None):
...
-                #  [0 3 1]]
-                print(out5.numpy())
-                #array([[[2],
-                #        [3],
-                #        [1]],
-                #       [[0],
-                #        [3],
-                #        [1]]])
+            paddle.disable_static()
+            data = np.array([[5,8,9,5],
```
Comment (XiaoguangHu01) [inline python/paddle/tensor/search.py:157]: x = paddle.to_tensor
to_tensor支持list的输入

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**